### DISCUSSION TITLE
template for organizational structure page

### DISCUSSION THREAD

**Discussion by ahmedabbous** [TDRA-ae/aegov-dls#18] [General]
is there any guidelines for the organizational structure page, as this page exist in most of the ministries and entities
example: https://www.fahr.gov.ae/Portal/en/about-fahr/organizational-structure.aspx.
should we use it as it is? or it need to be presented in different way? Thanks

**Comment by moiz-conceptlz** [comment]
Although it is useful information for many, the templates suggested on the design system were based on research from existing websites of ministries and authorities. 

It is also based on the fact that templates want to ensure all pages are mobile friendly, accessible and clear in terms of access to content on all devices.

Here are some examples of websites where organisation structure as a chart or content has not been created as a page. 

- https://mohap.gov.ae/
- https://www.moe.gov.ae/
- https://www.mofa.gov.ae/

There are certainly websites that showcase an organisation chart, while some prefer to call it the "The Team". The design system does recommend utilising the block ["Team"](https://designsystem.gov.ae/docs/blocks/team) to create consistent looking images of team members who are key contributors within the entity.

In any case, adding a page that showcases the organisation structure is very much a decision of the entity, and if there is value in adding that then you are free to create this page as creatively as required. **Ensure the page is mobile responsive, and content of the page is readable clearly on a mobile device.** 

Anyone can create a page with content that is necessary to the entity. Templates are guides that show the best method to organise content that is generic to everyone.
